src/server.cpp

```diff
@@ -2722,6 +2767,82 @@ void Server::UpdateCrafting(Player* player)
 	plist->changeItem(0, preview);
 }
 
+bool Server::handleChat(const std::string &name, const std::wstring &wname,
+	const std::wstring &wmessage, std::wstring &answer_to_sender,
+	u16 peer_id_to_avoid_sending)
+{
+	// If something goes wrong, this player is to blame
+	RollbackScopeActor rollback_scope(m_rollback,
+		std::string("player:") + name);
```

Comment:
Sorry, didn't realize the format of the rollback actor was to delimit with a colon and no space.